js/bybit.js

```diff
@@ -49,8 +49,9 @@ module.exports = class bybit extends Exchange {
                 'fetchDepositAddresses': false,
```

Comment:
@sc0Vu can't we update `fetchOrder`?
![image](https://user-images.githubusercontent.com/43336371/220956814-7afb1b5e-9c85-4e3f-87bb-76b40b6077f0.png)

Reply:
They didn't support spot market in v5 order history api.